pkg/lib/cockpit-components-dialog.jsx

```diff
@@ -299,7 +301,7 @@
         // the input focus from whatever element has it, which is
         // unpleasant and also disrupts the tests.
         if (rootElement.offsetParent)
-            ReactDOM.render(<Dialog {...dialogObj.props} />, rootElement);
+            root.render(<Dialog {...dialogObj.props} />);
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17939-20221121-134727-dbe868f6-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)